ietf/static_src/css/iab.scss

```diff
@@ -158,6 +162,12 @@
     }
 }
 
+@include media-breakpoint-up(xl) {
+    .nav-item--iab {
+        max-width: none;
+    }
+}
+
```

Comment:
Media queries should go inside the selector, i.e.
```
.nav-item--iab {
  [styles]
  
  @include media-breakpoint-up(xl) {
     max-width: none;
   }
}
```

Reply:
(when using sass, which you are)

Reply:
Thanks, I figured there was some better syntax for this